examples/lang/delegator/build-all.sh

```diff
@@ -1,6 +0,0 @@
-#!/bin/bash
```

Comment:
Hmm ... do we have a proper solution for this?

Reply:
We could simply leave the `build-all.sh` script but invoking `cargo contract build` instead of the local `build.sh` scripts.